src/env.h

```diff
@@ -1073,6 +1076,8 @@ class Environment {
   std::unordered_map<std::string, uint64_t> performance_marks_;
 
   bool has_run_bootstrapping_code_ = false;
+  bool has_serialized_options_ = false;
+
   bool can_call_into_js_ = true;
   Flags flags_;
   uint64_t thread_id_;
```

Comment:
There is still other places in the code that reference `process._breakFirstLine`, in `Module#_compile` for example.

Reply:
That is not affected by this PR, `process._breakFirstLine` will be set later during pre-execution according to the value of `getOptionValue('--inspect-brk')` which is already patched because we make sure we patch the options early enough before those are serialized in JS land.